modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSetContext.tsx

```diff
@@ -91,6 +92,7 @@ const FrontendDataSetContext = React.createContext({
 	loadData: () => {},
 	onActionDropdownItemClick: () => {},
 	onBulkActionItemClick: () => {},
+	onCreationActionClick: () => {},
```

Comment:
Just to keep naming consistent, can we call this `onCreationActionItemClick`, adding `Item`?